src/lib/logging/binary_recoverer.cpp

```diff
@@ -153,6 +153,7 @@ void BinaryRecoverer::recover() {
   }
 
   _update_transaction_id(last_transaction_id);
+  return _loaded_tables_amount;
```

Comment:
I'd guess you're talking about the number of tables loaded, not the amount?

http://grammarist.com/usage/amount-number/